File: mycollab-web/src/main/scala/com/mycollab/module/project/event/StandUpEvent.scala
package com.mycollab.module.project.event

import com.mycollab.events.ApplicationEvent

/**
  * @author MyCollab Ltd.
  * @since 5.0.3
  */
object StandUpEvent {

  class GotoList(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class DisplayStandupInProject(source: AnyRef, data: Integer) extends ApplicationEvent(source, data) {}

}

File: mycollab-web/src/main/java/com/mycollab/module/project/event/StandUpEvent.kt
package com.mycollab.module.project.event

import com.mycollab.events.ApplicationEvent

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
class StandUpEvent {
    class GotoList(source: Any, data: Any?) : ApplicationEvent(source, data)

    class DisplayStandupInProject(source: Any, data: Int) : ApplicationEvent(source, data)
}